### PR TITLE
Removed OPSWAT Medadefender Cloud support - no one uses that.

### PR DESCRIPTION
**Pull request by extrafu**
# Description
Removed OPSWAT Medadefender Cloud support. No one uses that as it is inefficient and cumbersome to setup with the Suricata MD5 plugin.

# Impacts
None, because no one uses it. If you do, drop me a mail and I'll send you a puppet.

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## Enhancements
* Removed OPSWAT Metadefender Cloud support

# UPGRADE file entries
To remove any left over meta defender configuration run /usr/local/pf/addons/upgrade/to-8.2-rm-opswat-metadefender.pl

### CODE REVIEW

**Review by jrouzierinverse — COMMENTED** [review]
Are these still needed?
- lib/pf/provisioner/opswat.pm
- html/captive-portal/templates/provisioner/opswat.html
- html/pfappserver/lib/pfappserver/Form/Config/Provisioning/opswat.pm
- docs/provisioner/opswat.asciidoc

**Comment by extrafu** [comment]
No, it's not about the provisioner, it's about the metadefender cloud stuff.

**Comment by julsemaan** [comment]
Cleanup looks good, we're just missing a script to cleanup the outdated configuration, @jrouzierinverse will take the lead on this and update the PR, we can merge after

File: addons/upgrade/to-8.2-rm-opswat-metadefender.pl [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+to-8.2-pfdetect-conf.pl
```
Comment (julsemaan) [inline addons/upgrade/to-8.2-rm-opswat-metadefender.pl]: This is wrong, and the script should be called 'to-8.3-rm-opswat-metadefender.pl'

**Comment by jrouzierinverse** [comment]
Done

**Pull request merged**